projects/canopy/src/lib/alert/docs/guide.stories.mdx

```diff
@@ -69,7 +69,7 @@ A basic inline message that should only be used if the others are unsuitable. Th
 
 ## Role
 
-The role attribute is automatically set to 'alert' for any non-info and generic variants. It can be manually set or overridden through property binding.
+It can be manually set through property binding or by passing the "none" value, preventing it from being set altogether. Otherwise, the role attribute is automatically assigned as 'alert' for any non-info and generic variants.
```

Comment:
To me this currently reads as !info && generic, rather than !info && !generic.

```suggestion
It can be manually set through property binding or by passing the "none" value, preventing it from being set altogether. Otherwise, the role attribute is automatically assigned as 'alert' for anything other than `info` and `generic` variants.
```